docs/deploy.rst

```diff
@@ -0,0 +1,244 @@
+.. _deploy:
+
+
+=========================
+Deploy your first cluster
+=========================
+
+This page shows you how to:
+
+- Create an organization
+- Set a region in Croud (determines where your data is stored)
+- Create a project
+- Check for available CrateDB products in that region
+- Deploy your first CrateDB cluster
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+.. NOTE::
+
+  This document assumes that you're `signed in`_ to your CrateDB Cloud user
+  account.
+
+
+Create an organization
+----------------------
+
+To begin with, let's create an organization. Organizations hold projects
```

Comment:
drop the use of "let's" for this type of documentation. only really applicable for blog post type "code together" content. here you are instructing the user (you're not doing it with them)